adserver/reports.py

```diff
@@ -286,7 +286,8 @@ def calculate_totals(self):
     def get_index_display(self, index):
         """Handle making sure dates use the same 3 letter syntax as Django."""
         if isinstance(index, datetime.date):
-            return index.strftime("%b %d, %Y")
+            # %a is the day of week, so you know which are weekends
+            return index.strftime("%b %d, %Y (%a)")
```

Comment:
Just a note that this cascades down all the way into the CSV exports. It also doesn't apply to advertisers (only publishers). I'm mostly OK with it but just wanted to air these notes.

Reply:
Yea -- not sure the best way to handle it. I think changing the CSS on the weekends is probably the best way to handle this (make them a slightly diff background color) -- but this seemed most explicit for now.